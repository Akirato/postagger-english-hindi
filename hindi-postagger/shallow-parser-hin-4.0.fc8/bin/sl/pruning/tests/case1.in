<Sentence id="1">
1	((	NP	
1.1	eka	QC	<fs af='eka,adj,any,any,,,any,'>|<fs af='eka,n,m,s,,0,,'>|<fs af='eka,n,m,p,,0,,'>|<fs af='eka,n,m,s,,1,,'>|<fs af='eka,n,m,p,,1,,'>
	))		
</Sentence>
