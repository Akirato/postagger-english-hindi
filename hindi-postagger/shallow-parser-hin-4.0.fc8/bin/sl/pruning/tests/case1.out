<Sentence id="1">
1	((	NP	
1.1	eka	QC	<fs af='eka,adj,any,any,,,any,'>
	))		
</Sentence>
